requirements/requirements-test-compatible.txt

```
@@ -0,0 +1,18 @@
+ase>=3.22.1
+cmake>=3.22.1
+codecov>=2.1.12
+coverage>=6.2
+cython>=0.29.26
+dynasor>=1.1.1; platform_system != "Windows"
+garnett>=0.7.1
+GitPython>=3.1.24
+gsd>=2.5.1
+matplotlib>=3.0.0
+numpy>=1.21.4
+pillow>=8.0.0 --only-binary=pillow
+pytest>=6.2.5
+pytest-cov>=3.0.0
+rowan>=1.3.0.post1
+scikit-build>=0.12.0
+scipy>=1.7.3
+sympy>=1.9
```

This should match https://github.com/glotzerlab/freud/blob/master/.circleci/ci-oldest-reqs.txt but with `>=`. We aim to support anything newer than our oldest pinned dependencies.
```suggestion
ase>=3.16.0
cmake>=3.12.0
codecov>=2.1.12
coverage>=6.2
cython>=0.29.14
dynasor>=1.1b0; platform_system != "Windows"
garnett>=0.7.1
GitPython>=3.1.24
gsd>=2.4.2
matplotlib>=3.0.0
numpy>=1.14.0
pillow>=6.2.0 --only-binary=pillow
pytest>=6.2.2
pytest-cov>=3.0.0
rowan>=1.2.1
scikit-build>=0.10.0
scipy>=1.1.0
sympy>=1.0
```